qa/backwards/shared/src/test/java/org/elasticsearch/test/ESBackcompatTestCase.java

```diff
@@ -212,17 +196,14 @@ private Settings addLoggerSettings(Settings externalNodesSettings) {
         return finalSettings.build();
     }
 
-    protected int minExternalNodes() { return 1; }
+    protected int minExternalNodes() {
+        return 1;
+    }
 
     protected int maxExternalNodes() {
         return 2;
     }
 
-    @Override
-    protected int maximumNumberOfReplicas() {
```

Comment:
Yeah - it really is. I imagine I could move maximumNumberOfReplicas into TestCluster and that'd be nicer.